examples/inverse/plot_dics_beamformer.py

```diff
@@ -53,7 +53,8 @@
 evoked = epochs.average()
 
 # Read forward operator
-forward = mne.read_forward_solution(fname_fwd, surf_ori=True)
+forward = mne.read_forward_solution(fname_fwd)
+forward = mne.convert_forward_solution(forward, surf_ori=True)
```

Comment:
Does `use_cps=True` work here (and in other examples)? Would be good to use it if we can. If we can't, let's do `use_cps=False` to avoid warnings.